PayPal/src/main/java/com/braintreepayments/api/paypal/PayPalClient.kt

```diff
@@ -193,20 +212,28 @@ class PayPalClient internal constructor(
 
             internalPayPalClient.tokenize(payPalAccount) { payPalAccountNonce: PayPalAccountNonce?, error: Exception? ->
                 if (payPalAccountNonce != null) {
+                    if (isAppSwitchFlow) {
+                        braintreeClient.sendAnalyticsEvent(
+                            PayPalAnalytics.HANDLE_RETURN_STARTED,
+                            analyticsParams
+                        )
+                    }
```

Comment:
I think this should also move to right at the start of this method (line 189)

Reply:
Updated: [209b57c](https://github.com/braintree/braintree_android/pull/1111/commits/209b57c44328035385fc5095f0077f5cdeaf59d4)